### PR TITLE
Enroll websites wtih tag

### PR DESCRIPTION
**Pull request by ykostov**

### CODE REVIEW

**Review by N-Patarov — APPROVED** [review]

**Review by AlenBG — APPROVED** [review]

File: lib/bgsite_official_web/controllers/tag_controller.ex [hunk 1]
```diff
@@ -28,9 +29,11 @@ defmodule BgsiteOfficialWeb.TagController do
 
   def show(conn, %{"id" => id}) do
     tag = Categories.get_tag!(id)
-    render(conn, "show.html", tag: tag)
+    website = websites = tag.websites |> Repo.preload(:tags)
```
Comment (rmilushev) [inline lib/bgsite_official_web/controllers/tag_controller.ex:32]: Oh, no... Please, make it:
```elixir
websites = tag.websites |> Repo.preload(:tags)
```
only.

File: lib/bgsite_official_web/controllers/tag_controller.ex [hunk 2]
```diff
@@ -28,9 +29,11 @@ defmodule BgsiteOfficialWeb.TagController do
...
+    render(conn, "show.html", tag: tag, websites_for_tag: website)
```
Comment (rmilushev) [inline lib/bgsite_official_web/controllers/tag_controller.ex:33]: then we will have :
```elixir
render(conn, "show.html", tag: tag, websites_for_tag: websites)
```

File: lib/bgsite_official_web/templates/tag/show.html.eex [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<div class="alert alert-info text-center">
+  <a href="/" class="blue-link-color"><%= @tag.title %>
+</div>
+<div class="container">
+  <div class="row">
+
+    <%= for websites <- @websites_for_tag do %>
```
Comment (rmilushev) [inline lib/bgsite_official_web/templates/tag/show.html.eex:7]: Please, keep the convention and change it to `website` as we have discussed a number of times.

File: lib/bgsite_official_web/templates/tag/show.html.eex [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+     <div class="card rounded" style="width: 18rem;">
+       <%=
+         link to: "https://#{websites.urls}", target: "_blank" do
+           img_tag BgsiteOfficial.BannerUploader.url(websites.banner, :thumb), class: "rounded card-img-top"
+         end
+         %>
+       <div class="card-body">
+
+         <!-- <%= for tag <- websites.tags do %>
+            <button type="button" class="btn btn-dark btn-sm my-1"> <%= tag.title %></button>
```
Comment (rmilushev) [inline lib/bgsite_official_web/templates/tag/show.html.eex:20]: I would suggest using the normal `link` definition with the class for buttons.

**Review by rmilushev — CHANGES_REQUESTED** [review]

**Review by rmilushev — APPROVED** [review]
Now it looks nice :+1:

**Pull request merged**